tests/globalkilltest/global_kill_test.go

```diff
@@ -271,11 +364,15 @@ func sleepRoutine(ctx context.Context, sleepTime int, conn *sql.Conn, connID uin
 		ch <- sleepResult{err: err}
 		return
 	}
-	defer rows.Close()
+	rows.Next()
```

Comment:
Why change this?

Reply:
Because calling Next will drive the result set to be processed, I think this is more robust and semantically correct.

Reply:
rows.Close() is called latter.